main.go

```diff
@@ -100,21 +102,31 @@ type AWS struct {
 	resourceMapping map[pb.ResourceType]string
 }
 
-func (a AWS) generateInput(script string, rt pb.ResourceType) *ec2.RunInstancesInput {
+func (a AWS) generateInput(script string, rt pb.ResourceType, labels []string) *ec2.RunInstancesInput {
 	instanceCount := int32(1)
 
+	imageId := a.imageID
+
+	for _, l := range labels {
```

Comment:
I'd like to suggest changing the implementation to use `strings.Contains` and `strings.TrimPrefix`.
This is to minimize the use of slices and thereby prevent the occurrence of panic.